src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -98,7 +98,7 @@ class AuthScreens extends React.Component {
         Pusher.init({
             appKey: CONFIG.PUSHER.APP_KEY,
             cluster: CONFIG.PUSHER.CLUSTER,
-            authEndpoint: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=AuthenticatePusher`,
+            authEndpoint: `${CONFIG.EXPENSIFY.PRIMARY_API_ROOT}api?command=AuthenticatePusher`,
```

Comment:
Should we use `ApiUtils.getRootApi` here or is this intentionally meant to always use the PRIMARY api? 

It was pointed out in slack that it's probably best to leave it out for the time being 
- Don't try to change pusher connection when API is switched
- Always use the primary api for the current environment for the pusher connection

Reply:
I am 👍 on leaving it out. There is maybe a very small chance that we would want to test the staging Pusher auth endpoint under any circumstance. The workaround would be to change the environment variables which is not hard to do on dev (but impossible on production).

Reply:
There's seems to be one thing that keeps evading us, perhaps it was more clear when I renamed `URL_API_ROOT` to `PROD_API_ROOT`:

`authEndpoint` is always given the PROD address 
- staging native apps don't use `.env.staging` but `.env.production` so they pass `www.expensify` here
- web/desktop use `.env.staging` but they also pass `www.expensify` because that's what's specified in `.env.staging` 

So perhaps it doesn't matter, or we're always supposed to pass the PROD _(AKA main, primary, non-staging)_ url here

Reply:
> > authEndpoint is always given the PROD address
> 
> yeah I think it's wrong. It should use the default API as we need to be able to test the Pusher auth endpoint locally on dev at least.

This is so. It uses either PROD or what you have in your .env configuration

What I mostly wanted to point out is that right now staging uses the prod URL 
#### Shouldn't the api here be `staging.exepensify.com`  when we're on staging? 
![image](https://user-images.githubusercontent.com/12156624/223668989-f4464c40-879b-455d-8209-ed3193f6f028.png)

Reply:
I can create a `ApiUtils.getDefaultApi()` that always returns the default endpoint for the given environment and update the code here

So that `authEndpoint` gets `staging.expensify` here (on staging env)

Reply:
I think we are both agreeing we don't need a change here. The default URL on staging is always production. 👍 It hits the production API by default. 👍 

This endpoint should be whatever is defined in the environment (not always production). Locally we can change it to be whatever we want.There's no need to be able to switch it on prod or staging versions of the app.